src/main/kotlin/com/github/biomejs/intellijbiome/BiomePackage.kt

```diff
@@ -17,44 +18,47 @@ class BiomePackage(private val project: Project) {
             return NodePackage.findDefaultPackage(project, "@biomejs/biome", interpreter)
         }
 
-    val configPath: String?
-        get() {
-            val settings = BiomeSettings.getInstance(project)
-            val configurationMode = settings.configurationMode
-
-            return when (configurationMode) {
-                ConfigurationMode.DISABLED -> null
-                ConfigurationMode.AUTOMATIC -> null
-                ConfigurationMode.MANUAL -> BiomeSettings.getInstance(project).configPath
-            }
+    fun configPath(file: VirtualFile): String? {
+        val settings = BiomeSettings.getInstance(project)
+        val configurationMode = settings.configurationMode
+        return when (configurationMode) {
+            ConfigurationMode.DISABLED -> null
+            ConfigurationMode.AUTOMATIC -> findPathUpwards(file, configValidExtensions.map { "$configName.$it" })?.path
```

Comment:
Biome already does that alone. If we don't pass any configuration path, Biome will try to resolve the configuration file by looking at the parent folders. Is there a specific reason why the plugin should do it?

Reply:
Even if Biome already does that in there are several reasons to do it here:
1. For LSP server proxy we should path config file. To find out which one we should run now, we should find config in plugin. This is because Biome doesn't support multiple configurations in LSP right now.
2. It seems that for IDEA with multiple projects it sounds reasonable that we should not enable biome, if there is no config.
3. In multiple project setup, you will need to run biome from proper working directory, so that `include`, `exclude` params from config will work correctly, as your config can probably not be in root folder then

Reply:
Thank you for the clarification. Is it worth documenting this behaviour in the `README.md`?

Reply:
will do

Reply:
done